src/lang/fr.json

```diff
@@ -0,0 +1,314 @@
+  "FSC.Configuration.Weekday.Name": "Nom du jour de la semaine",
+  "FSC.Configuration.Weekday.NameHelp": "Le nom textuel de ce jour de la semaine.",
+  "FSC.Configuration.Weekday.Add": "Ajouter un nouveau jour de la semaine",
+  "FSC.Configuration.Weekday.RemoveAll": "Supprimer tous les jours de la semaine",
+  "FSC.Configuration.Weekdays": "Jours de la semaine",
+  "FSC.Configuration.Weekday.Options": "Options des jours de la semaine",
+  "FSC.Configuration.Weekday.FirstWeekDay": "Démarrage du jour de la semaine",
+  "FSC.Configuration.Weekday.FirstWeekDayHelp": "Ceci définit le jour de la semaine où commence le premier jour de l'année 0.",
+  "FSC.Configuration.DefaultNoteVisibility": "Notez la visibilité par défaut du lecteur",
+  "FSC.Configuration.DefaultNoteVisibilityHint": "Pour les nouvelles notes, si par défaut l'option de visibilité du joueur est cochée ou non",
+  "FSC.Configuration.LeapYear.Title": "Paramètres des années bissextiles",
+  "FSC.Configuration.LeapYear.Rule": "Règle de l'année bissextile",
+  "FSC.Configuration.LeapYear.RuleHelp": "Les règles à suivre pour déterminer les années bissextiles.",
+  "FSC.Configuration.LeapYear.CustomMod": "Quand les années bissextiles arrivent",
+  "FSC.Configuration.LeapYear.CustomModHelp": "Chaque x nombre d'années est une année bissextile.",
+  "FSC.Configuration.LeapYear.Rules.None": "Aucun",
+  "FSC.Configuration.LeapYear.Rules.Gregorian": "Gregorian",
+  "FSC.Configuration.LeapYear.Rules.Custom": "Personnalisé",
+  "FSC.Configuration.LeapYear.Days": "Nombre de jours dans une année bissextile",
+  "FSC.Configuration.LeapYear.DaysHelp": "Le nombre de jours qui composent ce mois au cours d'une année bissextile.",
+  "FSC.Configuration.LeapYear.Options": "Options des années bissextiles",
+  "FSC.Configuration.Time.Title": "Paramètres de l'heure",
+  "FSC.Configuration.Time.HoursInDay": "Heures dans une journée",
+  "FSC.Configuration.Time.HoursInDayHelp": "Combien d'heures composent une seule journée.",
+  "FSC.Configuration.Time.MinutesInHour": "Minutes dans une heure",
+  "FSC.Configuration.Time.MinutesInHourHelp": "Combien de minutes composent une seule heure.",
+  "FSC.Configuration.Time.SecondsInMinute": "Secondes dans une minute",
+  "FSC.Configuration.Time.SecondsInMinuteHelp": "Combien de secondes composent une seule minute.",
+  "FSC.Configuration.Time.GameTimeRatio": " Secondes de jeu par secondes de vie réelle ",
+  "FSC.Configuration.Time.GameTimeRatioHelp": "Combien de secondes s'écoulent dans le jeu pour chaque seconde qui passe en temps réel.",
+  "FSC.Configuration.Time.Options": "Options de temps",
+  "FSC.Configuration.Time.ClockSettings": "Paramètres de l'horloge",
+  "FSC.Configuration.Time.UpdateFrequency": "Fréquence de mise à jour",
+  "FSC.Configuration.Time.UpdateFrequencyHelp": "Ceci détermine la fréquence à laquelle l'horloge se met à jour.",
+  "FSC.Configuration.Time.UnifyGameAndClock": "Unifier les Horloges Démarrer/Pause Avec la Pause du Jeu",
```

Comment:
This should probably be something like "Unifier le démarrage/pause de l'horloge avec l'horloge du jeu"